hack/tools/helm-cm/go.work

```diff
@@ -0,0 +1,3 @@
+go 1.22.1
```

Comment:
Whats this file for? Also does it support just the minor version like `go.mod`?
```suggestion
go 1.22
```

Reply:
it looks like we dont need it so im going to remove